cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -139,42 +161,55 @@ export default {
         return { content: contents[id], flipped: false };
       };
 
-      this.cards = Object.keys(contents).filter(_filter).map(_createCard);
+      this.credentials = Object.keys(contents).filter(_filter).map(_createCard);
```

Comment:
Why you need `_createCard` still?

Reply:
this is just rename  from this.cards to this.credentials

Reply:
I meant why you need `map(_createCard)`